src/StringTagField.php

```diff
@@ -384,7 +384,7 @@ public function setCanCreate($canCreate)
     public function performReadonlyTransformation()
     {
         $field = parent::performReadonlyTransformation();
-        $field->setValue(implode(', ', $this->Value()));
+        $field->setValue(implode(', ', $this->getFormattedValue()));
```

Comment:
Shouldn't we be setting the internal value of the readonly field to the internal value of this field?

Reply:
Updated